source/grammar/openpulse1.g4

```diff
@@ -0,0 +1,74 @@
+/**** ANTLRv4  grammar for OpenPulse1.0.
+
+The following grammar specifies valid OpenPulse statements within the .*? of
+OpenQASM3 below.
+
+calibrationDefinition
+    : 'defcal' calibrationGrammar? Identifier
+    ( LPAREN calibrationArgumentList? RPAREN )? identifierList
+    returnSignature? LBRACE .*? RBRACE  // for now, match anything inside body
+    ;
+
+specifically, `.*? -> body` below.
+
+****/
+
+grammar openpulse1;
+
+body
+    : (statement)+
+    ;
+
+statement
```

Comment:
See `statement` in https://github.com/Qiskit/openqasm/blob/46c734efc4039022e1c3fe02c48fbf2dd7d026f4/source/grammar/qasm3.g4#L32. Don't think you need the redefinition. I believe those statement types are the ones valid in `defcal`'s.

Reply:
You'll need to add `instruction` as well. Maybe can combine them into `pulseStatement`